engine-jupyter/templates/deployment.yaml

```diff
@@ -35,24 +48,34 @@ spec:
             httpGet:
               path: /
               port: {{ .Values.service.internalPort }}
+          volumeMounts:
```

Comment:
As comented above, I'd give the opportunity of not saving state if no volume is given and add the same condition:

```
{{- if .Values.notebooks.volumes }}
```